src/components/PostDetails/PostDetails.scss

```diff
@@ -7,6 +7,10 @@
     text-align: right;
   }
 
+  &__visiblelist {
+    display: none;
+  }
+
```

Comment:
Looks weird `display: none;` for a class `visiblelist`. Use kebeb-case (`.visible-list`)